src/daemon/http/server.rs

```diff
@@ -508,6 +515,49 @@ pub async fn metrics(req: Request) -> RoutingResult {
                 res.push_str(&format!("krill_cas_bgp_announcements_valid{{ca=\"{}\"}} {}\n", ca, nr));
             }
 
+            res.push('\n');
+            res.push_str("# HELP krill_ca_parent_success status of last ca-parent connection (0=issue, 1=success)\n");
+            res.push_str("# TYPE krill_ca_parent_success gauge\n");
+            for (ca, parent_statuses) in parent_stats.iter() {
+                if ca.as_str() != TA_NAME {
+                    for (parent, status) in parent_statuses.iter() {
+                        // skip the ones for which we have no status yet, i.e it was really only just added
+                        // and no attempt to connect has yet been made.
+                        if let Some(exchange) = status.last_exchange() {
+                            let value = if exchange.was_success() { 1 } else { 0 };
+                            res.push_str(&format!(
+                                "krill_ca_parent_success{{ca=\"{}\", parent=\"{}\"}} {}\n",
+                                ca, parent, value
+                            ));
+                        }
+                    }
+                }
+            }
+
+            res.push('\n');
+            res.push_str(
+                "# HELP krill_ca_parent_last_success_time timestamp of last successful ca-parent connection\n",
```

Comment:
What format is this timestamp in? Do we need to say something about the format or about the time zone?

Reply:
I believe that 'timestamp', unless specified otherwise, is generally understood to be seconds since 1/1/1970. But.. I will clarify the help texts to be clear on this - no harm in that!

Reply:
I've never come across that definition of timestamp before. I'm used to seeing seconds since 1/1/1970 as "epoch time" though.

Reply:
Changed it to "unix timestamp in seconds". (epoch time is wrong according to wikipedia because other epochs exist)